externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -195,8 +196,11 @@ object SparkSQLEngine extends Logging {
         s" and submitted at $submitTime.")
     } else {
       var spark: SparkSession = null
+      val checkFlag = new AtomicBoolean(false)
```

Comment:
we can define it as global, and the name should be refine, maybe like `sparkSessionCreated`

Reply:
> we can define it as global, and the name should be refine, maybe like `sparkSessionCreated`

thanks, i will do that